api/tests.py

```diff
@@ -94,6 +143,50 @@ def test_get_one_demo(self):
         self.assertEqual(response["token"], payload["token"])
         self.assertEqual(response["status"], payload["status"])
 
+    def test_get_one_demo_without_id_and_userid(self):
+        self.setUp2()
+        payload = self.demo
+        payload2=self.demo2  #new demo is created so now there are 2 different demo of 2 diff users
+
+        url = '/api/demo/' + str(None) + '/' + str(None)
+        responses = self.client.get(url)
+        # It returns a list containing all the demos preset in the database
+        response = json.loads(responses.content.decode('utf-8'))[0] #returns 1st demo config
+        self.assertEqual(response["name"], payload["name"])
+        self.assertEqual(response["id"], payload["id"])
+        self.assertEqual(response["user_id"], payload["user_id"])
+        self.assertEqual(response["address"], payload["address"])
+        self.assertEqual(response["description"], payload["description"])
+        self.assertEqual(response["footer_message"], payload["footer_message"])
+        self.assertEqual(response["cover_image"], payload["cover_image"])
+        self.assertEqual(response["terminal"], payload["terminal"])
+        self.assertEqual(response["token"], payload["token"])
+        self.assertEqual(response["status"], payload["status"])
+
+        response2 = json.loads(responses.content.decode('utf-8'))[1]  #return second user config
+        self.assertEqual(response2["name"], payload2["name"])
+        self.assertEqual(response2["id"], payload2["id"])
+        self.assertEqual(response2["user_id"], payload2["user_id"])
+        self.assertEqual(response2["address"], payload2["address"])
+        self.assertEqual(response2["description"], payload2["description"])
+        self.assertEqual(response2["footer_message"], payload2["footer_message"])
+        self.assertEqual(response2["cover_image"], payload2["cover_image"])
+        self.assertEqual(response2["terminal"], payload2["terminal"])
+        self.assertEqual(response2["token"], payload2["token"])
+        self.assertEqual(response2["status"], payload2["status"])
+        
+
+    def test_demo_not_found(self):
```

Comment:
Here i have used the  user_id of first user(payload) and id of the demo of the second user(payload2), Therefore it should return "Nor Found".